skabase/SKALogger/test/SKALogger_test.py

```diff
@@ -98,10 +96,8 @@ def test_SetCentralLoggingLevel(self, tango_context, setup_log_test_device,
         device_details = []
         device_details.append(levels)
         device_details.append(targets)
-        print("device_details: ", device_details)
```

Comment:
instead of removing the print, why don't you print them in the logger as info message?

Reply:
The print message was left by mistake. We really dont need it so I removed it.